tests/strategy/test_strategy.py

```diff
@@ -65,13 +66,22 @@ def test_load_strategy_invalid_directory(result, caplog, default_conf):
 
 
 def test_load_not_found_strategy(default_conf):
-    strategy = StrategyResolver(default_conf)
+    default_conf['strategy'] = 'NotFoundStrategy'
     with pytest.raises(OperationalException,
                        match=r"Impossible to load Strategy 'NotFoundStrategy'. "
                              r"This class does not exist or contains Python code errors."):
+        strategy = StrategyResolver(default_conf)
         strategy._load_strategy(strategy_name='NotFoundStrategy', config=default_conf)
```

Comment:
@xmatthias Are you sure the resolver should be used (and tested) this way?
`_load_strategy()` was already called in the `__init__()` of `StrategyResolver`, the strategy is (should be) loaded and available as `strategy.strategy`

Reply:
you're right, the exception is raised in the first line.

95299d94c4c824eed2c3aca6fc370a8b85633901 solves this.